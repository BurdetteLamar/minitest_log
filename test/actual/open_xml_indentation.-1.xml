<log><section_ name='Section'><data_ class='Integer' method=':to_s' name='indentation'>-1</data_></section_></log>
