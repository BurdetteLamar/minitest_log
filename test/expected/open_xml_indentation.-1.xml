<log><summary errors='0' failures='0' verdicts='1'/><section name='Section'><data class='Integer' method=':to_s' name='indentation'>-1</data></section><verdict id='__error_count__' method='verdict_assert_equal?' outcome='passed'><exp_value>0</exp_value><act_value>0</act_value></verdict></log>
